action/protocol/staking/stakingpb/staking.proto

```diff
@@ -20,7 +20,7 @@ message BucketIndices {
 }
 
 message Delegate {
-    bytes owner = 1;
+    string owner = 1;
     string address = 2;
     string rewardAddress = 3;
```

Comment:
nit: to be space efficient, it is ok to store owner, address, rewardadress as bytes.

Reply:
yes, but lots of proto used string to store, so just use string